fhirr4/ballerina/src/main/resources/fhirservice/http_service_builder.bal

```diff
@@ -48,6 +49,13 @@ isolated function getHttpService(Holder h, r4:ResourceAPIConfig apiConfig) retur
                         }
                         fhirContext = check r4:getFHIRContext(ctx);
                         executeResourceResult = executeWithID(id, fhirContext, fhirService, resourceMethod);
+                        if executeResourceResult is r4:Bundle {
+                            if paginationEnabled {
+                                executeResourceResult = addPagination(fhirContext, executeResourceResult, req.extraPathInfo);
+                            } else {
+                                executeResourceResult = populateBundleTotal(executeResourceResult);
+                            }    
+                        }
```

Comment:
can you move these repetitive code to a function?

Reply:
moved